app/assets/stylesheets/helpers/_header.scss

```diff
@@ -341,3 +341,47 @@
     margin-right: .5em;
   }
 }
+
+
+.show-global-bar #global-header-bar {
+  display: none;
+}
+
+#global-bar {
+  margin-bottom: 15px;
+  padding: 15px 0;
+  background-color: $light-blue-25;
+  display: none;
+
+  .show-global-bar & {
+    display: block;
+  }
+
+  .global-bar-message-container {
+    @include core-19;
+    @extend %site-width-container;
+    position: relative;
+
+    @include media(tablet) {
+      .dismiss {
+        position: absolute;
+        right: 0;
+        top: 0;
+      }
+    }
+  }
+
+  .global-bar-message {
+    margin-top: 0;
+    margin-bottom: 0;
+
+    @include media(tablet) {
+      max-width: 66.67%;
+    }
+
+    strong {
```

Comment:
This should be a class rather than styling on the element. Suggest `global-bar-message-title`?

Reply:
Done in https://github.com/alphagov/static/pull/1583/commits/3fdc23e8df288099f364658e2445577790b59d67